modules/tinymce/CHANGELOG.md

```diff
@@ -9,13 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - New `expand` function added to `tinymce.selection` which expands the selection around the nearest word. #TINY-9001
 - New `expand` function added to `tinymce.dom.RangeUtils` to return a new range expanded around the nearest word. #TINY-9001
-- New `color_map_background` and `color_map_foreground` option which sets the base colors used in the `backcolor` and `forecolor` toolbar buttons and menu items. #TINY-9184
+- New `color_map_background` and `color_map_foreground` options which set the base colors used in the `backcolor` and `forecolor` toolbar buttons and menu items. #TINY-9184
```

Comment:
It probably doesn't make it any clearer, but should we put *respectively* in there?

Reply:
Normally I'd say yes, but because we are describing 4 items (backcolor and forecolor, each with a toolbar button and menu item) I think it would get too wordy. We can't squeeze _respectively_ in the middle, it would have to go at the end, and then what is it respective of?

Reply:
> Normally I'd say yes, but because we are describing 4 items (backcolor and forecolor, each with a toolbar button and menu item) I think it would get too wordy. We can't squeeze respectively in the middle, it would have to go at the end, and then what is it respective of?

FWIW, this was almost literally my thinking when I was doing the copy-edit and trying to make _respectively_ work in the edit that existed before I went with the edit above.